styleguide/placeholder.scss

```diff
@@ -74,3 +83,47 @@
     transition: 50ms fill ease-out;
   }
 }
+
+.kiln-placeholder .placeholder-add-component {
```

Comment:
This could be combined into .kiln-placeholder on L6? Might be easier to read if SASS were  organized into three blocks: `.kiln-placeholder`, `.kiln-permanent-placeholder`, and `.kiln-page-area`